scripts/cli/src/tests/15_portfolio.ts

```diff
@@ -106,7 +97,16 @@ async function main(): Promise<void> {
     )
   );
 
-  
+  console.log(`Election Status: ${await api.query.staking.eraElectionStatus()}`);
+  await checkEraElectionClosed();
+  console.log(`Election Status: ${await api.query.staking.eraElectionStatus()}`);
+  // AddAPortfolioManager is not possible because of old permission format
+  console.log("Portfolio: StopStakingAPortion");
+  await unbond(dave, 100);
+
+  console.log("Portfolio: StartStakingANewOperator");
+  await nominate(dave, alice.publicKey);
+
```

Comment:
Why is staking done in the "Portfolio" test?

Reply:
I didn't write the original portfolio test, but I assume to add more content to the test itself.